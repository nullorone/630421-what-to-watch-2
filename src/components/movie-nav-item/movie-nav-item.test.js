import React from "react";
import renderer from "react-test-renderer";
import MovieNavItem from "./movie-nav-item";

describe(`Make snapshot`, () => {
  const initProps = {
    active: false,
    text: `My link, bro`
  };

  it(`Get snapshot component`, () => {
    const tree = renderer
      .create(
          <MovieNavItem {...initProps}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
