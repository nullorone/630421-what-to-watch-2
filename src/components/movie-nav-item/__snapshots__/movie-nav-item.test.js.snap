// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make snapshot Get snapshot component 1`] = `
<li
  className="movie-nav__item"
>
  <a
    className="movie-nav__link"
    data-tab-name="my link, bro"
    href="#"
    onClick={[Function]}
  >
    My link, bro
  </a>
</li>
`;
